docs/nconnect-build-from-source-code.md

```diff
@@ -0,0 +1,109 @@
+---
+id: nconnect-build-from-source
+title: Build nConnect from Source
+---
+
+## Preparation
+
+*If you already have git and golang compiler on your computer, skip to the next section.*
+
+To build nConnect from source, we need **git** tool and **go** compiler.
+If your PC or laptop doesn't have git and go installed, please use the following link to install them:
+
+### Install **git**
+- For Windows:
+Download and install git for Windows:
+```
+https://git-scm.com/download/win
+```
+
+- Debian/Ubuntu
+For the latest stable release for your version of Debian/Ubuntu
+```
+$ sudo apt update
+$ sudo apt install git
+```
+
+### Install **Go**.
+You can download the **Go** release from this URL:
+```
+https://go.dev/dl/
+```
+
+After downloading, please follow this URL to install **Go**.
+```
+https://go.dev/doc/install
+```
+
+## Build nConnect on Windows 
+
+Download the nConnect source code from Github:
+
+- Download nConnect source code
+
+```
+> git clone https://github.com/nknorg/nconnect
+```
+
+- Build nConnect
+
+```
+> cd nConnect
+nConnect> go build -o nConnect.exe bin/main.go
```

Comment:
It's better to use `make`. Some arguments are missing this way

Reply:
Yes, modified. We suggest building it by make, but if they don't install make utilities on Windows, then use "go build* as an alternative.

Reply:
I don't think we ever need to support without Makefile. It's part of our building dependencies if users decide to build from source, just like git

Reply:
Okay, let's removed it.